spec/integration/rspec/include_matcher_spec.rb

```diff
@@ -258,6 +258,46 @@
             in_color(color_enabled)
         end
       end
+
+      it "produces the correct failure message when fuzzy matching" do
```

Comment:
Thanks for this. Would you mind also adding a test in the `context "that is large"` section as well? (This section tests what happens when the size of the object is so large that the "Expected ... not to include ..." part of the error message has to be split up into multiple lines.)

Reply:
Done!